test/test_core.py

```diff
@@ -5942,6 +5942,10 @@ def test_fs_writev(self):
     self.set_setting('FORCE_FILESYSTEM')
     self.do_runf('fs/test_writev.c', 'success')
 
+  def test_fs_writev_partial_write(self):
+    self.set_setting('FORCE_FILESYSTEM')
```

Comment:
Is this line needed?

Reply:
It isn't. I'll remove in my next commit.